Userland/DevTools/Playground/main.cpp

```diff
@@ -165,6 +165,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
                 GUI::MessageBox::show(window, "Unable to save file.\n", "Error", GUI::MessageBox::Type::Error);
                 return;
             }
+            window->set_modified(false);
```

Comment:
It seems like we need to explicitly mark as "not modified" after we save, otherwise the window ends up in a perpetual loop of not closing on save.